sway/config/output.c

```diff
@@ -391,6 +391,17 @@ void apply_output_config_to_outputs(struct output_config *oc) {
 	}
 }
 
+void reset_outputs(void) {
+	struct output_config *oc = NULL;
+	int i = list_seq_find(config->output_configs, output_name_cmp, "*");
```

Comment:
What about non-wildcard configs?

Reply:
The wildcard config is only ever applied directly when there is no output specific config. If there is an output specific config, that is applied instead.

https://github.com/swaywm/sway/blob/77a3694a782f48ea476958bf97e3f82ebafae0b1/sway/config/output.c#L375-L381

Reply:
Ah, I see